### ISSUE TITLE
External links containing id are broken

### ISSUE THREAD

**Issue by dGilli** [verbb/vizy#75]
**Description**
External Links containing "#" output incorrectly.

**Steps to reproduce**
1. Insert external link (ex. "https://www.google.com#bla")
2. Looks like url output is some sort of object ```<a href="{bla:url||https://www.google.com}" target="">Link</a>```

**Additional info**
- Plugin version: 1.0.7
- Craft version: 3.7.14
- Multi-site: Yes

**Comment by engram-design** [comment]
Looks like this was an issue with our parsing logic for ref tags. Should be fixed for the next release. To get the fix early, change your `verbb/vizy` requirement in `composer.json` to:

```
"require": {
  "verbb/vizy": "dev-craft-3 as 1.0.7",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.0.8](https://github.com/verbb/vizy/releases/tag/1.0.8).
